core/tauri-codegen/src/embedded_assets.rs

```diff
@@ -73,6 +74,14 @@ impl EmbeddedAssets {
       .map(Self)
   }
 
+  #[cfg(debug_assertions)]
+  /// A dummy EmbeddedAssets for use during development builds.
+  /// Compressing + including the bytes of assets during development takes a long time.
+  /// On development builds, assets will simply be resolved & fetched from the configured dist folder.
+  pub fn new(_: &Path) -> Result<Self, EmbeddedAssetsError> {
+    Ok(EmbeddedAssets(HashMap::new()))
+  }
+
```

Comment:
we can change this logic to a `if cfg!` in the `context.rs` file to create the file system assets struct if debug_assertions are enabled